src/lib/comms/sol-oic-client.c

```diff
@@ -1056,7 +1113,7 @@ request_new(enum sol_coap_method method, enum sol_coap_msgtype type, struct sol_
     }
 
     r = _set_token_and_mid(request->base.pkt, &request->token);
-    SOL_INT_CHECK_GOTO(r, < 0, error;);
+    SOL_INT_CHECK_GOTO(r, < 0, error; );
```

Comment:
The new version of uncrustify must want this, because this fails my git pre-commit hook if the space is not there. Whatever, I'll still force-push it.